src/main/kotlin/blackjack/view/BlackJackController.kt

```diff
@@ -0,0 +1,38 @@
+package blackjack.view
+
+import blackjack.domain.BlackJack
+import blackjack.domain.CardDeck
+import blackjack.domain.MutableCards
+import blackjack.domain.Player
+import blackjack.domain.PlayerName
+import blackjack.domain.Players
+
+class BlackJackController {
+    fun initPlayers(): Players {
+        val players =
+            Players(
+                players =
+                    inputPlayerNameView().map {
+                        Player(PlayerName(it), MutableCards(cards = mutableListOf()))
```

Comment:
Player는 처음에 아무런 카드를 가지고있지 않을텐데요. 
[default arguments](https://kotlinlang.org/docs/functions.html#default-arguments)를 활용해서 아래와 같이 변경해보면 어떨까요? 

```suggestion
                        Player(PlayerName(it))
```